docs/aws_sso.md

```diff
@@ -0,0 +1,56 @@
+If you are working with AWS, there are two things you should avoid at all costs:
+
+* Use account’s root credentials for day-to-day work
+* Provision individual users in your account and generate credentials for them
+
+AWS Single Sign On (SSO) with Athenz allows you to avoid both. Because an Athenz
+domain is directly related to an AWS account, the same authorization mechanism
+can be leveraged for SSO to the AWS console, via SAML-based integration. The
+Identity Provider service validates the user and then uses Athenz
+to determine the AWS role(s) the user can assume. This set of roles also indirectly
+identifies all the accounts the user can access from the console. In its simplest
+form, the single (potentially large) list can be presented to AWS in a SAML
+document, and AWS allows selection of the account/role to use.
+
+## Onboard to AWS
+
+To register an AWS Account with a domain, run the following command:
```

Comment:
Replaced Onboarding to AWS text with this. Original content:
"...AWS account creation is managed by the Public Cloud Team and you can request
 one by visiting [Cosmo](https://cosmo.ouroath.com)."